File: sdks/ruby/lib/novabilling/charges/types/create_charge_dto_charge_model.rb
# frozen_string_literal: true

module Novabilling
  module Charges
    module Types
      module CreateChargeDtoChargeModel
        extend Novabilling::Internal::Types::Enum

        STANDARD = "STANDARD"
        GRADUATED = "GRADUATED"
        VOLUME = "VOLUME"
        PACKAGE = "PACKAGE"
        PERCENTAGE = "PERCENTAGE"
      end
    end
  end
end

File: sdks/swift/Sources/Schemas/CreateChargeDtoChargeModel.swift
import Foundation

public enum CreateChargeDtoChargeModel: String, Codable, Hashable, CaseIterable, Sendable {
    case standard = "STANDARD"
    case graduated = "GRADUATED"
    case volume = "VOLUME"
    case package = "PACKAGE"
    case percentage = "PERCENTAGE"
}